### DISCUSSION TITLE
There is any wait to show the notification on multiple monitors with this lib?

### DISCUSSION THREAD

**Discussion by tatosjb** [hoodie/notify-rust#168] [Q&A] [answered]
I have an issue with this. On ubuntu, the notification only shows up on one of the monitors. Trying to solve that, I find this repository.

My question is, using this lib, can I place a notification wherever I want? I was not able to find this in the docs.

**Comment by hoodie** [comment, marked as answer]
Which display the actual notification is shown on is up to implementation that ultimately draws the toast. This library does not do this, it sends a command to your desktop's display manager (KDE, Gnome, Xfce etc) according to a specification.
There is a [position hint in the spec](https://docs.rs/notify-rust/latest/notify_rust/enum.Hint.html#variant.X), but I don't think that any implementation currently actually obeys this.
